modules/iot-core/README.md

```diff
@@ -0,0 +1,112 @@
+# Google Cloud IoT Core Module
+
+This module setup Cloud IoT Core Registry, register IoT Devices and configure Pub/Sub topics required in Cloud IoT Core.
```

Comment:
"This module sets up a Cloud IoT Core Registry, registers IoT Devices and configures Pub/Sub topics required in Cloud IoT Core."